src/python/libseccomp.pxd

```diff
@@ -117,6 +117,13 @@ cdef extern from "seccomp.h":
         int32_t error
```

Comment:
Sorry to be a nuisance here, but as written in the current patchset, a `git bisect` would fail on the commit before this.

In the prior commit you added test 63 via its `*.tests` file.  The `regression` script will also expect the python test at that time and thus fail

You'll probably want to refactor your commits as follows:
1. The commit that adds the C functionality to libseccomp
2. A commit that adds Python functionality to src/python
3. The C and Python tests and associated Makefile.am changes